torchtune/models/llama3_2/_model_builders.py

```diff
@@ -67,6 +72,8 @@ def llama3_2_3b(
         scale_factor=32,
         tie_word_embeddings=tie_word_embeddings,
     )
+
```

Comment:
nonsense

Reply:
these clearly should have been here already, no?